pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -1,18 +1,22 @@
 { stdenv, fetchgit, zlib, libpng, qt4, pkgconfig
-, withGamepads ? true, SDL # SDL is used for gamepad functionality
+, withGamepads ? true, SDL ? null # SDL is used for gamepad functionality
```

Comment:
Well, there is no reason to change here, I just used to do that way, as I do in Bochs expression.

Anyway, it doesn't fail to find SDL...